.github/actions/setup/action.yml

```diff
@@ -13,11 +13,6 @@ inputs:
 runs:
   using: "composite"
   steps:
-    # Set default Python to python 3.x, and set Python path such that pip install works properly
-    - uses: actions/setup-python@v5
-      with:
-        python-version: '3.x'
```

Comment:
> The python installed
by setup-python (which is CPython) seems to have issues running WPT.

<https://github.com/web-platform-tests/wpt/issues/44427>

WPT currently needs 3.11, rather than 3.12

Reply:
That's not actually the issue we were getting - WPT was not getting the packages we installed with pip (and then just churned exceptions for 5 hours)

Reply:
huh. that's unfortunate. Perhaps our run.sh should create a venv and have its own requirements.txt? big meh though, python envs are hard enough as-is :)

Reply:
Yeah venv will be _the way_ ™️ (or whatever the kids are using in current year)